src/jit/unwindarm.cpp

```diff
@@ -1780,7 +1783,7 @@ void UnwindInfo::InitUnwindInfo(Compiler* comp, emitLocation* startLoc, emitLoca
     // However, its constructor needs to be explicitly called, since the constructor for
     // UnwindInfo is not called.
 
-    uwiFragmentFirst.UnwindFragmentInfo::UnwindFragmentInfo(comp, startLoc, false);
+    uwiFragmentFirst.Init(comp, startLoc, false);
```

Comment:
Once again, what is the problem with this form?

Reply:
btw, we do have placement new. E.g.,:
```
new (&newLvaTable[i], jitstd::placement_t()) LclVarDsc();
```

Reply:
I saw this but couldn't figure out how to use it. Especially the second parameter. Do you have a suggestion for it?

Reply:
> Once again, what is the problem with this form?

GCC doesn't allow direct constructor calls.

Reply:
Maybe:
```
new (&uwiFragmentFirst, jitstd::placement_t()) UnwindFragmentInfo(comp, startLoc, false);
```
?

I believe the `jitstd::placement_t()` is just a trick to force using our "placement new" overload.

Reply:
thanks, let's see how this will work. Placement new is the right choice here.